actionview/test/template/translation_helper_test.rb

```diff
@@ -164,7 +164,8 @@ def test_translate_with_html_count
     assert_equal "<a>Other &lt;One&gt;</a>", translate(:'translations.count_html', count: "<One>")
   end
 
-  def test_translation_returning_an_array_ignores_html_suffix
+  def test_translate_marks_array_of_translations_with_a_html_safe_suffix_as_safe_html
+    assert_equal true, translate(:'translations.array_html').all? { |element| element.html_safe? }
```

Comment:
We generally prefer Ruby's definition of true and wouldn't explicitly assert it as the return value.

Think this would be clearer as:

```ruby
translate(:'translations.array_html').tap do |translated|
  assert_equal %w( foo bar ), translated
  assert translated.all?(&:html_safe?)
end
```

Reply:
All the other tests I've seen on this file use `assert_equal true` so I supposed it was preferred. I will update it.